src/sass/pages/room/_amenities.scss

```diff
@@ -0,0 +1,131 @@
+.amenities {
+  background-color: $main-bg-color;
+}
+
+.amenities-items {
+  list-style-type: none;
+  margin: 16px 0 0;
+  padding: 0;
+}
+
+.amen-main-header {
+  font-family: 'Times New Roman', Times, serif;
+  text-transform: uppercase;
+  font-weight: 400;
+  font-size: 32px;
+  line-height: 1.15;
+  color: $black;
+  margin-bottom: 16px;
+}
+
```

Comment:
add 
```
.main-description {
margin-bottom: 32px;
}
```

Reply:
1. added mb to main-description
2. changed paddings to margins in items
3. fixed svg 'right arrow' sizes